### ISSUE TITLE
Sparql Store does not compute correct Node Names in XML generation

### ISSUE THREAD

**Issue by PonteIneptique** [Capitains/Nautilus#66]
```python
from capitains_nautilus.cts.resolver import SparqlAlchemyNautilusCTSResolver
from tests.cts.config import sqlite_address

resolver = self.SparqlAlchemyNautilusCTSResolver(
    ["./tests/testing_data/latinLit2"], 
    sqlalchemy_address=sqlite_address
)
resolver.parse()
for x in resolver.graph.namespaces():
     print(x)
>>> ('xml', rdflib.term.URIRef('http://www.w3.org/XML/1998/namespace'))
>>> ('rdf', rdflib.term.URIRef('http://www.w3.org/1999/02/22-rdf-syntax-ns#'))
>>> ('rdfs', rdflib.term.URIRef('http://www.w3.org/2000/01/rdf-schema#'))
>>> ('xsd', rdflib.term.URIRef('http://www.w3.org/2001/XMLSchema#'))
>>> ('', rdflib.term.URIRef('http://chs.harvard.edu/xmlns/cts/'))
>>> ('dts', rdflib.term.URIRef('http://w3id.org/dts-ontology/'))
>>> ('tei', rdflib.term.URIRef('http://www.tei-c.org/ns/1.0/'))
>>> ('skos', rdflib.term.URIRef('http://www.w3.org/2004/02/skos/core#'))
>>> ('cpt', rdflib.term.URIRef('http://purl.org/capitains/ns/1.0#'))

# but
print(
    "http://chs.harvard.edu/xmlns/cts/textgroup",
    resolver.graph.namespace_manager.qname("http://chs.harvard.edu/xmlns/cts/textgroup")
)
>>> http://chs.harvard.edu/xmlns/cts/textgroup ns1:textgroup
```

**Comment by PonteIneptique** [comment]
From what I understand, the binding might not be registered in SQLite because the prefix is empty.

**Comment by PonteIneptique** [comment]
The bug is seemingly due to a limitation of the SQLAlchemy store : https://github.com/RDFLib/rdflib-sqlalchemy/issues/37

**Comment by PonteIneptique** [comment]
Closed by #68
